### ISSUE TITLE
Does library supports load data from from torch.utils.data.Dataset?

### ISSUE THREAD

**Issue by KennyTC** [KevinMusgrave/pytorch-metric-learning#370]

**Comment by KennyTC** [comment]
The problem is already solved, so closed it.
